arbitrum-docs/launch-orbit-chain/how-tos/manage-fee-collectors.md

```diff
@@ -0,0 +1,306 @@
+    functionName: 'getL1RewardRate',
+});
+```
+
+To set a new `L1RewardRecipient` address, you can call the method `setL1PricingRewardRecipient(address)` of the [ArbOwner](/build-decentralized-apps/precompiles/02-reference.md#arbowner) precompile, and pass the address of the new reward recipient. For example:
+
+```shell
+cast send --rpc-url $ORBIT_CHAIN_RPC --private-key $OWNER_PRIVATE_KEY 0x0000000000000000000000000000000000000070 "setL1PricingRewardRecipient(address) ()" $NEW_L1REWARDRECIPIENT_ADDRESS
+```
+
+Alternatively, you can use the Orbit SDK to set the new address:
+
+```typescript
+const owner = privateKeyToAccount(<OwnerPrivateKey>);
+const orbitChainClient = createPublicClient({
+    chain: <OrbitChainDefinition>,
+    transport: http(),
+}).extend(arbOwnerPublicActions);
+
+const transactionRequest = await orbitChainClient.arbOwnerPrepareTransactionRequest({
+    functionName: 'setL1PricingRewardRecipient',
+    args: [<NewL1RewardRecipientAddress>],
+    upgradeExecutor: false,
+    account: owner.address,
+});
+
+await orbitChainClient.sendRawTransaction({
+    serializedTransaction: await owner.signTransaction(transactionRequest),
+});
+```
+
+To change the reward rate, you can use the method `setL1PricingRewardRate(uint64)` of the [ArbOwner](/build-decentralized-apps/precompiles/02-reference.md#arbowner) precompile and pass the amount of wei per gas unit to reward. For example:
+
+```shell
+cast send --rpc-url $ORBIT_CHAIN_RPC --private-key $OWNER_PRIVATE_KEY 0x0000000000000000000000000000000000000070 "setL1PricingRewardRate(uint64) ()" $NEW_REWARD_RATE
+```
+
+Or using the Orbit SDK:
+
+```typescript
+const owner = privateKeyToAccount(<OwnerPrivateKey>);
+const orbitChainClient = createPublicClient({
+    chain: <OrbitChainDefinition>,
+    transport: http(),
+}).extend(arbOwnerPublicActions);
+
+const transactionRequest = await orbitChainClient.arbOwnerPrepareTransactionRequest({
+    functionName: 'setL1PricingRewardRate',
+    args: [<NewRewardRate>],
+    upgradeExecutor: false,
+    account: owner.address,
+});
+
+await orbitChainClient.sendRawTransaction({
+    serializedTransaction: await owner.signTransaction(transactionRequest),
+});
+```
+
+## How to use the fee distribution contracts?
+
+For now, we've described how to set the individual collector addresses for each of the fee types. Some chains may require to have multiple addresses receive the collected fees of any of the available types. For those cases, there's the possibility of using a distributor contract that can gather all fees of a specific type and distribute those among multiple addresses.
+
+This section shows how to configure a distributor contract to manage the fees of a specific type.
+
+:::info Distributor contracts currently not supported in the Orbit SDK
+
+Currently, the Orbit SDK doesn't support deploying and configuring distribution contracts, but will soon be added. So for now, this section will only show examples using `cast send`.
```

Comment:
```suggestion
Currently, the Orbit SDK doesn't support deploying and configuring distribution contracts, but it will soon be added. So, for now, this section will only show examples using `cast send`.
```